app/models/settings/general.rb

```diff
@@ -128,9 +128,19 @@ class General < RailsSettings::Base
     # Push Notifications
     field :push_notifications_ios_pem, type: :string
 
+    # Feed
+    field :feed_pinned_article_id, type: :integer, validates: { article_id: true, allow_nil: true }
```

Comment:
I really, really wanted to avoid adding a table for this :-D. I was actually going to implement this in Redis, as it's the perfect place for a globally unique value, but then I thought about using `Settings::General`...

I know it's not the perfect place semantically but the table isn't either. I wish we had a secondary persistent storage for these kind of things...

Reply:
How about a middle ground? Store the setting in `Settings::General` but add a domain model like the one I suggested to `app/models` so we get (a) nicer code and (b) an easy way out of `Settings::General` if we ever add a secondary storage.

Reply:
Great! I can do that :-) I'm definitely in favor of proxying objects for future proofing!